app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ReviewsController < ApplicationController
+
+  before_action :require_user
+
+  def create
+    @video = Video.find(params[:video_id])
+    review = Review.new review_params.merge!(video_id: params[:video_id], user: current_user)
```

Comment:
okay, it's clear. in case that I'm using @video.reviews.build... I have to reload the reviews in unhappy path. Is it a "good practice"? (I don't want to hope that unhappy path is accessed frequently but if so it hits the database every time which I think is avoidable)